contrib/pgbase36/Trunk.toml

```
@@ -0,0 +1,24 @@
+[extension]
+name = "pgbase36"
+extension_name = "base36"
+version = "0.0.1"
```

Let's checkout v1.0.0 (above) and use that version here
```suggestion
version = "1.0.0"
```